### PR TITLE
Added UKBB and open targets data source documentation

### PR DESCRIPTION
**Pull request by hlnicholls**
In this PR I have had a go at writing some bios for some of the data sources. Please let me know what you think (or if potentially I've put the text in the wrong markdown file)

The updated markdowns are:
- `docs/python_api/datasource/open_targets/_open_targets.md`
- `docs/python_api/datasource/ukbiobank/_ukbiobank.md`

The text is taken and updated partly from already existing descriptions in https://genetics-docs.opentargets.org/ and https://platform-docs.opentargets.org/target

### CODE REVIEW

File: docs/python_api/datasource/open_targets/_open_targets.md [hunk 1]
```diff
@@ -7,3 +7,9 @@
     display: none;
   }
 </style>
+
+The Open Targets Platform is a comprehensive resource that aims to aggregate and harmonise various types of data to facilitate the identification, prioritisation, and validation of drug targets. Genomic data from Open Targets provides gene identification and genomic coordinates that are integrated into the gene index of our ETL pipeline.
```
Comment (ireneisdoomed) [inline docs/python_api/datasource/open_targets/_open_targets.md:11]: I'd include the text that goes from "Genomic data from Open Targets..." to "... for a reviewed protein product." as docstrings inside the `OpenTargetsTarget` class. This extract basically describes the target, not OT as a source.

**Review by ireneisdoomed — CHANGES_REQUESTED** [review]
Thank you for the suggestions!

File: docs/python_api/datasource/open_targets/_open_targets.md (hunk 1 above)
Comment (hlnicholls) [reply docs/python_api/datasource/open_targets/_open_targets.md]: Thank you for this feedback! I've moved this text to the `OpenTargetsTarget` class docstring and I have updated the rest of the text I've got in this `_open_targets.md` file to focus more on OT as a source. I've still included a few sentences focusing on the genomic data, but happy to rewrite as needed. 

Also, I realised I'm not sure how much we want to write here so I'm also happy to expand if this is too short.

**Review by ireneisdoomed — APPROVED** [review]

**Comment by ireneisdoomed** [comment]
Changes make sense. We can build from here.

**Pull request merged**